test/controllers/user_controller_test.rb

```diff
@@ -3,6 +3,10 @@
 class UserControllerTest < ActionController::TestCase
   api_fixtures
 
+  setup do
```

Comment:
We've always used `def setup` in the past. Is this way the typical way to do it now? Basically I'd prefer to standardise one way or the other and I'd just like to figure out which is more normal so we can standardise on that.